src/Akeneo/Pim/Enrichment/Bundle/Command/CleanRemovedAttributesFromProductAndProductModelCommand.php

```diff
@@ -52,6 +54,8 @@ class CleanRemovedAttributesFromProductAndProductModelCommand extends Command
     private CountProductsAndProductModelsWithInheritedRemovedAttributeInterface $countProductsAndProductModelsWithInheritedRemovedAttribute;
     private RouterInterface $router;
     private string $pimUrl;
+    private GetAllBlacklistedAttributeCodesInterface $getAllBlacklistedAttributeCodes;
```

Comment:
In the construct it can be null
```suggestion
    private ?GetAllBlacklistedAttributeCodesInterface $getAllBlacklistedAttributeCodes;
```